### PR TITLE
add two versions into emailappusageversionsusercounts

### PR DESCRIPTION
**Pull request by haizeie**

### CODE REVIEW

File: api-reference/beta/api/reportroot-getemailappusageversionsusercounts.md [hunk 1]
```diff
@@ -70,6 +70,8 @@ The CSV file has the following headers for columns.
 - Outlook 2007
 - Undetermined
 - Report Period
+- Outlook M365
```
Comment (Lauragra) [inline api-reference/beta/api/reportroot-getemailappusageversionsusercounts.md:73]: Is this accurate? Per branding guidelines, we don't abbreviate Microsoft 365 as M365.

File: api-reference/beta/api/reportroot-getemailappusageversionsusercounts.md [hunk 2]
```diff
@@ -120,7 +122,7 @@ Follow the 302 redirection and the CSV file that downloads will have the followi
 HTTP/1.1 200 OK
 Content-Type: application/octet-stream
 
-Report Refresh Date,Outlook 2016,Outlook 2013,Outlook 2010,Outlook 2007,Undetermined,Report Period
+Report Refresh Date,Outlook 2016,Outlook 2013,Outlook 2010,Outlook 2007,Undetermined,Report Period,Outlook M365,Outlook 2019
```
Comment (Lauragra) [inline api-reference/beta/api/reportroot-getemailappusageversionsusercounts.md:125]: Same here, not sure of the M365.

File: concepts/changelog.md [hunk 3]
```diff
@@ -13,6 +13,12 @@ For a summary of the value of these API changes, as well as recent tools, compon
 
 ## September 2020
 
+### Reports API
```
Comment (Lauragra) [inline concepts/changelog.md:16]: Change this to "Reports".

File: api-reference/beta/api/reportroot-getemailappusageversionsusercounts.md (hunk 1 above)
Comment (haizeie) [reply api-reference/beta/api/reportroot-getemailappusageversionsusercounts.md:73]: Yes, we decide to use this name "Outlook M365" in this PR: https://microsoftgraph.visualstudio.com/onboarding/_git/onboarding/pullrequest/3929

File: api-reference/beta/api/reportroot-getemailappusageversionsusercounts.md (hunk 2 above)
Comment (haizeie) [reply api-reference/beta/api/reportroot-getemailappusageversionsusercounts.md:125]: The same above

**Pull request merged**